Algorithm.CSharp/AddOptionContractExpiresRegressionAlgorithm.cs

```diff
@@ -131,13 +131,13 @@ public override void OnData(Slice data)
             {"Loss Rate", "50%"},
             {"Win Rate", "50%"},
             {"Profit-Loss Ratio", "0.92"},
-            {"Alpha", "-0.023"},
-            {"Beta", "0.005"},
+            {"Alpha", "-0.021"},
+            {"Beta", "-0.01"},
             {"Annual Standard Deviation", "0.006"},
             {"Annual Variance", "0"},
-            {"Information Ratio", "-3.424"},
-            {"Tracking Error", "0.057"},
-            {"Treynor Ratio", "-4.775"},
+            {"Information Ratio", "-3.374"},
+            {"Tracking Error", "0.058"},
+            {"Treynor Ratio", "2.133"},
```

Comment:
Surprising these changed with no increase in the number of trades?

Reply:
Yeah, this is because the option exercise happens immediately in the PR while in master it crosses into the next day, so the daily performance samples of the algorithm have a 1-day shift when the option is exercised 👍